src/app/articles/articles-list/article-pre-view/article-pre-view.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="articlePreView">
+  <img class="articlePreView__title-image" src="{{article.titleImage}}" alt="title-image">
+  <div class="articlePreView__title">{{article.title}}</div>
+  <div class="author">
+    <img class="author__profile-picture" src="{{article.authorImage}}" alt="profile-picture">
+
```

Comment:
Nice markup!
1. Do you want to add spaces in the interpolations ike `{{ abc }}`?
2. Remove this empty line?